src/Bridge/MoodlePlugin.php

```diff
@@ -197,6 +197,18 @@ public function hasFilesWithName($pattern): bool
         return count($result) !== 0;
     }
 
+    /**
+     * Determine if the plugin has a (phpunit.xml) configuration file.
+     *
+     * @return bool
+     */
+    public function hasPhpUnitConfig(): bool
+    {
+        $result = $this->getFiles(Finder::create()->depth(0)->name('phpunit.xml'));
```

Comment:
I guess phpunit can be in any directory (otherwise it might be enough to do `file_exists($this->pluginDir . '/phpunit.xml')`)?

Reply:
Yeah, I initially was thinking about to allow them in any subdir, but then realised that we only use those `phpunit.xml` files when they are on plugin base/root dir. And I never have seen them elsewhere.

https://github.com/moodlehq/moodle-plugin-ci/blob/master/src/Command/PHPUnitCommand.php#L105-L115

So I think it's ok to prevent recursion, we only care about the base/root one, that in fact is the one created by our `admin/tool/phpunit/cli/init.php --buildcomponentconfigs` option (the one we are preventing here to be used if there is already a `phpunit.xml` file created by the dev.

Reply:
About using simpler `file_exists()` vs `Finder` component, I saw that there were uses of both in the class and decided to go to the later, just to know/experiment more about it. There isn't much else to say. I'm happy either way.

Reply:
I suggest to use `file_exists` for better readability.

Reply:
Have amended it to use simpler `is_file($this->directory . '/phpunit.xml')`